spanner/ddl/ast.go

```diff
@@ -388,13 +391,19 @@ func (agc AutoGenCol) PrintAutoGenCol() string {
 	if agc.Name == constants.UUID && agc.GenerationType == "Pre-defined" {
 		return " DEFAULT (GENERATE_UUID())"
 	}
+	if agc.GenerationType == constants.SEQUENCE {
+		return fmt.Sprintf(" DEFAULT (GET_NEXT_SEQUENCE_VALUE(SEQUENCE %s)) ", agc.Name)
```

Comment:
How does this work? Will we allow user to write a custom `SEQUENCE`?

Reply:
<img width="882" alt="Screenshot 2024-05-27 at 2 29 47 PM" src="https://github.com/GoogleCloudPlatform/spanner-migration-tool/assets/35952883/5266ddc9-1fa0-4e77-b846-6c4f2965b5c8">
<img width="529" alt="Screenshot 2024-05-27 at 2 29 19 PM" src="https://github.com/GoogleCloudPlatform/spanner-migration-tool/assets/35952883/0101ab24-fb2c-4d3b-a100-87706bb31e8c">

yes, they will be able to enter the values required to configure the sequence as shown in this UI